build.rs

```diff
@@ -222,7 +223,7 @@ const ASM_TARGETS: &[AsmTarget] = &[
         preassemble: true,
     },
     AsmTarget {
-        oss: &[WINDOWS],
+        oss: &[WINDOWS, UEFI],
```

Comment:
This deserves a comment about why UEFI uses Windows ASM stuff unlike everything else.